src/plugins/home/public/application/components/home_app.js

```diff
@@ -54,41 +54,44 @@ export function HomeApp({ directories, solutions }) {
     savedObjectsClient,
     getBasePath,
     addBasePath,
-    environmentService,
+    // environmentService,
     telemetry,
   } = getServices();
-  const environment = environmentService.getEnvironment();
-  const isCloudEnabled = environment.cloud;
+  // const environment = environmentService.getEnvironment();
+  // const isCloudEnabled = environment.cloud;
 
-  const renderTutorialDirectory = (props) => {
-    return (
-      <TutorialDirectory
-        addBasePath={addBasePath}
-        openTab={props.match.params.tab}
-        isCloudEnabled={isCloudEnabled}
-      />
-    );
-  };
+  // const renderTutorialDirectory = (props) => {
+  //   return (
+  //     <TutorialDirectory
+  //       addBasePath={addBasePath}
+  //       openTab={props.match.params.tab}
+  //       isCloudEnabled={isCloudEnabled}
+  //     />
+  //   );
+  // };
 
-  const renderTutorial = (props) => {
-    return (
-      <Tutorial
-        addBasePath={addBasePath}
-        isCloudEnabled={isCloudEnabled}
-        getTutorial={getTutorial}
-        replaceTemplateStrings={replaceTemplateStrings}
-        tutorialId={props.match.params.id}
-        bulkCreate={savedObjectsClient.bulkCreate}
-      />
-    );
-  };
+  // const renderTutorial = (props) => {
+  //   return (
+  //     <Tutorial
+  //       addBasePath={addBasePath}
+  //       isCloudEnabled={isCloudEnabled}
+  //       getTutorial={getTutorial}
+  //       replaceTemplateStrings={replaceTemplateStrings}
+  //       tutorialId={props.match.params.id}
+  //       bulkCreate={savedObjectsClient.bulkCreate}
+  //     />
+  //   );
+  // };
```

Comment:
**question** Do we want to keep this commented code or should we remove it instead?